react-ui/src/components/LogInForm.js

```diff
@@ -28,21 +33,30 @@ class logInForm extends Component {
 
 
   login = event => {
-    let user_id = this.state.loginParams.user_id;
-    let user_password = this.state.loginParams.user_password;
-    if (user_id === "admin" && user_password === "123") {
-      localStorage.setItem("token", "T");
-      this.setState({
-        islogged: true
-      });
-    }
+    let email = this.state.loginParams.email;
+    let password = this.state.loginParams.password;
+    var params = {email: email, password: password}
+    axios.get(`/api/login/?${queryString.stringify(params)}`)
+    .then(res => {
+      console.log(res.data)
+      if(res.data){
+        this.setState({
+          firstName : res.data.firstName,
```

Comment:
I would recommend using the studentId or workerId instead of first name since it's possible for multiple users to have the same first name